audit/src/handlers/index.ts

```diff
@@ -10,29 +10,13 @@ export type EventHandler<T extends object> = (
   ...args
 ) => (ev: Event<T>) => Promise<boolean>;
 
-export const ServiceStartedHandler = (auditDomain: AuditController) => async (
-  ev: Event<ServiceStartedPayload>,
-) => {
-  // Transform the event payload into an auditable event
-  const auditItem: AuditLogItem = {
-    id: v4(),
-    subject: `${ev.payload.name}-${ev.payload.type}`,
-    verb: 'STARTED',
-    entity: 'NONE',
-  };
-
-  logger.info('serviceStartedReceived', ev.payload);
-  return await auditDomain.recordAudit(auditItem);
-};
-
 export const UserLoggedInHandler = (auditDomain: AuditController) => async (
   ev: Event<UserLoggedInPayload>,
 ) => {
   const auditItem: AuditLogItem = {
-    id: v4(),
-    subject: `${ev.payload.name}-${ev.payload.email}`,
-    verb: 'LOGGED_IN',
-    entity: ev.payload.userId,
+    entity: `user:${ev.payload.userId}`,
+    action: 'LOGGED_IN',
+    object: 'application',
```

Comment:
Is this correct?

Reply:
Thinking about it we either need actions LOGIN-SUCCESS and LOGIN-FAILURE .... or an extra column `result` ... what do think?

Reply:
Not everything has failure or success outcomes.

Maybe LOGIN_ATTEMPT and LOGGED_IN

Reply:
I sort of thinking we need a `result` or `extradata` column - because there will often be additional data associated with the event ... so in this example it would be "true" or "false" in response to the "LOGGED_IN" event

Reply:
Also - in addition to the slack question -- can we pull off the source of the event and have this as the `object` ?

Reply:
ok i'm guessing we can add a json result column. how do send/get the source of the event?